src/test/java/ejercicio/banco/service/AccountServiceImplTest.java

```diff
@@ -75,4 +75,33 @@ public void testFindAccountReturnsNullIfAccountNotFoundExceptionIsThrown() {
 
         assertNull(account);
     }
+
+
+    @Test
+    public void testFindAccountsCallsFindFromRepository() {
+        accountService.findAccounts(FILE_NAME, "NAME");
```

Comment:
Duplicated "NAME" constant. Refactor to a `private static final String ACCOUNT_NAME = "NAME";` (for example).